src/SwedbankPay.Sdk/Payments/VippsPayments/VippsPayment.cs

```diff
@@ -0,0 +1,45 @@
+﻿using swedbankpay.Sdk.Payments.VippsPayments;
+using SwedbankPay.Sdk.Extensions;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+
+namespace SwedbankPay.Sdk.Payments.VippsPayments
+{
+    public class VippsPayment
+    {
+        private VippsPayment(VippsPaymentResponse paymentResponse, HttpClient client)
+        {
+            PaymentResponse = paymentResponse.Payment;
+            var operations = new VippsPaymentOperations(paymentResponse.Operations, client);
+            Operations = operations;
+        }
+
+        public VippsPaymentOperations Operations { get; }
+
+        public PaymentResponseObject PaymentResponse { get; }
+
+
+        internal static async Task<VippsPayment> Create(VippsPaymentRequest paymentRequest,
```

Comment:
Hm, same here as in MobilePay. Is this how `Create` is implemented in the other payment instruments?